src/coreclr/gc/gcinterface.h

```diff
@@ -914,6 +914,9 @@ class IGCHeap {
     // Registers a frozen segment with the GC.
     virtual segment_handle RegisterFrozenSegment(segment_info *pseginfo) = 0;
 
+    // Updates given frozen segment
+    virtual void UpdateFrozenSegment(segment_handle seg, uint8_t* allocated, uint8_t* committed) = 0;
```

Comment:
this new method needs to go to the end of the interface so we don't break backward compatibility with older version of coreclr.dll from clrgc.dll